docs/rules/README.md

```diff
@@ -8,266 +8,265 @@ Some rules are fixable using the `--fix` command line flag. Those rules are mark
-* [brace-style](brace-style.md): enforce one true brace style
-* [camelcase](camelcase.md): require camel case names
-* [comma-spacing](comma-spacing.md): enforce spacing before and after comma (fixable)
-* [comma-style](comma-style.md): enforce one true comma style
+* [array-bracket-spacing](array-bracket-spacing.md): enforce consistent spacing inside array brackets (fixable)
+* [block-spacing](block-spacing.md): enforce consistent spacing inside single-line blocks (fixable)
+* [brace-style](brace-style.md): enforce consistent curly brace style for blocks
+* [camelcase](camelcase.md): enforce camelcase naming convention
+* [comma-spacing](comma-spacing.md): enforce consistent spacing before and after commas (fixable)
+* [comma-style](comma-style.md): enforce consistent comma style
 * [computed-property-spacing](computed-property-spacing.md): require or disallow padding inside computed properties (fixable)
 * [consistent-this](consistent-this.md): enforce consistent naming when capturing the current execution context
-* [eol-last](eol-last.md): enforce newline at the end of file, with no multiple empty lines (fixable)
-* [func-names](func-names.md): require function expressions to have a name
-* [func-style](func-style.md): enforce use of function declarations or expressions
-* [id-blacklist](id-blacklist.md): disallow certain identifiers to prevent them being used
-* [id-length](id-length.md): enforce minimum and maximum identifier lengths (variable names, property names etc.)
-* [id-match](id-match.md): require identifiers to match the provided regular expression
-* [indent](indent.md): specify tab or space width for your code (fixable)
-* [jsx-quotes](jsx-quotes.md): specify whether double or single quotes should be used in JSX attributes (fixable)
-* [key-spacing](key-spacing.md): enforce spacing between keys and values in object literal properties
-* [keyword-spacing](keyword-spacing.md): enforce spacing before and after keywords (fixable)
-* [linebreak-style](linebreak-style.md): enforce linebreak style (fixable)
-* [lines-around-comment](lines-around-comment.md): enforce empty lines around comments
-* [max-depth](max-depth.md): specify the maximum depth that blocks can be nested
-* [max-len](max-len.md): specify the maximum length of a line in your program
-* [max-nested-callbacks](max-nested-callbacks.md): specify the maximum depth callbacks can be nested
-* [max-params](max-params.md): specify the number of parameters that can be used in the function declaration
-* [max-statements](max-statements.md): specify the maximum number of statement allowed in a function
-* [max-statements-per-line](max-statements-per-line.md): specify the maximum number of statements allowed per line
-* [new-cap](new-cap.md): require a capital letter for constructors
-* [new-parens](new-parens.md): disallow the omission of parentheses when invoking a constructor with no arguments
+* [eol-last](eol-last.md): enforce a single newline at the end of files (fixable)
```

Comment:
That's one of the things I tried to keep consistent across rule descriptions ("newline" vs "newlines" vs etc.). In this case it's specific, and I think "at least one newline" is the clearest and most accurate.